app-backend/backsplash-server/src/main/scala/com/rasterfoundry/backsplash/services/WmsService.scala

```diff
@@ -74,13 +74,20 @@ class WmsService[LayerReader: OgcStore](layers: LayerReader, urlPrefix: String)(
                 .parMapN {
                   case (Valid(mbTile), hists) =>
                     logger.debug(s"Style: ${layer.style}, hists are: ${hists}")
+                    // We pad with this invisible tile to ensure tile sizes are as expected
                     val invisiTile =
                       mbTile.prototype(params.width, params.height)
                     val fullTile = invisiTile merge mbTile
-                    val tileResp = layer.style map {
-                      _.renderImage(fullTile, params.format, hists)
-                    } getOrElse {
-                      Render(fullTile, layer.style, params.format, hists)
+                    val tileResp = fullTile.bandCount match {
```

Comment:
it's probably worth adding a TODO here referencing the fact that we should handle this differently in the future by using the default layer color mode instead of a generic "rgb" mapping.